openshift/ci-operator/update-ci.sh

```diff
@@ -44,8 +44,7 @@ $CURDIR/generate-ci-config.sh knative-$VERSION 4.6 true false > ${CONFIG}__46.ya
 $CURDIR/generate-ci-config.sh knative-$VERSION 4.7 true false > ${CONFIG}__47.yaml
 $CURDIR/generate-ci-config.sh knative-$VERSION 4.8 true false > ${CONFIG}__48.yaml
 $CURDIR/generate-ci-config.sh knative-$VERSION 4.9 true false > ${CONFIG}__49.yaml
-$CURDIR/generate-ci-config.sh knative-$VERSION 4.10 true false > ${CONFIG}__410.yaml
-$CURDIR/generate-ci-config.sh knative-$VERSION 4.11 true false true > ${CONFIG}__411-tls.yaml
+$CURDIR/generate-ci-config.sh knative-$VERSION 4.11 false true true > ${CONFIG}__411.yaml
 
```

Comment:
It's looking better now. But 4.10 is completely missing here.
According to https://docs.google.com/document/d/1s0UrrCDh3b7TrqbKU0X7D62L7AccyxqtjfoxeiZ2vXo/edit#heading=h.btnn1nimzwdy the Serverless 1.25 release should still support 4.6 but Serverless 1.26 will support OCP 4.8 and newer.
I'm wondering if you want to drop 4.6, 4.7 from here and add 4.10 back. 
On a general note. It might be sufficient to test on the oldest and newest. So, in theory having 4.8 and 4.11 should be sufficient. But I will leave it to you. Testing all versions from 4.6 to 4.11 sounds like an overkill.
WDYT?

Reply:
Sure, sounds good to me. Updated.
We will need to update https://github.com/openshift/release as well.